cirq-rigetti/cirq_rigetti/aspen_device.py

```diff
@@ -224,23 +225,23 @@ def __repr__(self):
         return f'cirq_rigetti.RigettiQCSAspenDevice(isa={self.isa!r})'
 
     def _json_dict_(self):
-        return {'isa': self.isa.to_dict()}
+        return {'isa': json.loads(self.isa.json())}
 
     @classmethod
     def _from_json_dict_(cls, isa, **kwargs):
-        return cls(isa=InstructionSetArchitecture.from_dict(isa))
+        return cls(isa=InstructionSetArchitecture.from_raw(json.dumps(isa)))
 
 
-@_provide_default_client  # pragma: no cover
+# pragma: no cover
```

Comment:
The `#  pragma: no cover` comment is ineffective here.
Please move to the end of the line 238, ie, the end of the `def` statement to
ignore coverage for the entire `get_rigetti_qcs_aspen_device` function.

Better yet, adding a unit test would be nicer, even with a mock of the `get_instruction_set_architecture` function.